cores/esp32/HardwareSerial.h

```diff
@@ -86,6 +86,10 @@ class HardwareSerial: public Stream
     {
         return write((uint8_t) n);
     }
+    void setBaudRate(uint32_t baud) 
```

Comment:
Isn't it same as https://github.com/espressif/arduino-esp32/pull/2223 which is already merged?

Reply:
Yes, it appears to be a duplicate. Instead of calling it "updateBaudRate" this PR is calling it "setBaudRate"